components/test_pd/src/mocker/service.rs

```diff
@@ -238,6 +238,7 @@ impl PdMocker for Service {
             .insert(region_id, req.get_leader().clone());
 
         let mut resp = RegionHeartbeatResponse::default();
+        resp.set_region_id(req.get_region().get_id());
```

Comment:
why?

Reply:
It returns the the default region not the requested the region.